clickbench/README.md

```diff
@@ -0,0 +1,52 @@
+<!---
```

Comment:
I think that we should move this file to `benchmarks/clickbench/README.md` so that all the benchmarks are in one place.